msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -9,26 +9,36 @@ ms.topic: how-to
 # Device permissions for the browser
 
 > [!NOTE]
-> The change to how device permissions are handled in the browser is currently available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md) only. 
-> This change will be generally available (GA) by January 21, 2022.
+> The new feature on how device permissions are handled in the browser are currently available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
```

Comment:
Please refer to the existing docs. We do not use the term "feature". Please revert.